spec/helpers/session.rb

```diff
@@ -0,0 +1,18 @@
+module SessionHelpers
+  def sign_up(user)
+    visit '/users/new'
+    fill_in :name, with: user.name
+    fill_in :username, with: user.username
+    fill_in :email, with: user.email
+    fill_in :password, with: user.password
+    fill_in :password_confirmation, with: user.password_confirmation
+    click_button 'Sign up'
+  end
+
+  def sign_in(email:, password:)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `email`. You can also write as `sign_in(*)` if you want the method to accept any arguments but don't care about them.<br>Lint/UnusedMethodArgument: Unused method argument - `password`. You can also write as `sign_in(*)` if you want the method to accept any arguments but don't care about them.